accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,78 @@
+# ***Display Trusted Owners in PM UI***
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Donnie Goodson ([donnie-msft](https://github.com/donnie-msft))
+- GitHub Issue https://github.com/NuGet/Home/issues/12501
+
+## Summary
+
+When available, highlight in the Packages Manager UI (PM UI) that the owner is trusted by the package source with a NuGet package icon. 
+In the case of `nuget.org`, also provide a hyperlink to the owner's profile page (ie, `https://www.nuget.org/profiles/<owner name>`).
+A Trusted Owner is defined as an owner that's indicated by a package source's [Search API](https://learn.microsoft.com/nuget/api/search-query-service-resource#search-result), as opposed to the field in `.nuspec`.
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+
+Customers need a clear understanding of who created a NuGet package in order to trust that package.
+The term "owner" has become overloaded as NuGet has evolved, which has caused confusion. 
+The "author" metadata is also misleading as it's a free-form text field which doesn't imply any credentials.
+
+## Explanation
+
+### Functional explanation
```

Comment:
Will the details pane (right side of the Browse tab, after clicking a result) show owners and or authors still?

I know most of the stuff on the right comes from registration which doesn't return owners but you could plump the owners data from the search result into the right pane to display purposes.

Reply:
Was thinking to add "Owner(s)" above "Author(s)", probably with the same profile links as packages list.

Reply:
Might be good to have a ❔ icon next to the authors or even owners to describe the difference on hover. Don't know if this is part of the VS UI language but there could be prior are of how to describe nuanced topics inline of UI.

Seems good otherwise

Reply:
Will look into how to add a help link in development, but imagining the ℹ️ icon will do the trick.